extensions/ql-vscode/test/vscode-tests/minimal-workspace/local-queries/local-databases.test.ts

```diff
@@ -763,18 +766,9 @@ describe("local databases", () => {
       });
     });
 
-    describe("when codeQL.codespacesTemplate is set to false", () => {
-      it("should not create a skeleton QL pack", async () => {
-        jest.spyOn(Setting.prototype, "getValue").mockReturnValue(false);
-
-        await databaseManager.openDatabase(mockDbItem.databaseUri);
-        expect(createSkeletonPacksSpy).toBeCalledTimes(0);
-      });
-    });
-
-    describe("when codeQL.codespacesTemplate is not set", () => {
+    describe("when codeQL.codespacesTemplate is set to false or not defined", () => {
```

Comment:
I've combined these, since `isCodespacesTemplate` [can't return `undefined`](https://github.com/github/vscode-codeql/blob/02c1d7ef9ed03402f78021c3161cc3f74f3dcb1e/extensions/ql-vscode/src/config.ts#L641). It would get converted to `false` anyway.